fontra2fontir/src/fontra.rs

```diff
@@ -13,6 +14,24 @@ use fontir::error::Error;
 use serde::Deserialize;
 use write_fonts::types::Tag;
 
+#[derive(Debug, Clone, Deserialize, Hash, PartialEq, Eq, PartialOrd, Ord)]
+pub(crate) struct AxisName(pub(crate) String);
+
+impl Display for AxisName {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.write_str(self.0.as_str())
+    }
+}
```

Comment:
if we *were* going to go this route, I think it would also be useful to implement the conversion traits (`From`) as well as probably `AsRef<str>`.